### DISCUSSION TITLE
Value for reporting_entity_type Does Not Require Allowed Values?

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#33] [Q&A] [answered]
The value for the schema field found in all 3 files of reporting_entity_type has a type of String and does not provide in the definition a set of Allowed values, instead the field definition says

"The type of entity that is publishing the machine-readable file (a group health plan, health insurance issuer, or a third party with which the plan or issuer has contracted to provide the required information, such as a third-party administrator, a health care claims clearinghouse, or a health insurance issuer that has contracted with a group health plan sponsor)."

Does this imply that this field can be filled with any variation of strings like the above examples found in parentheses text for type of entity? Or should a set of allowed fields be provided for this field to use in the schema definition of the field?

**Comment by shaselton-usds** [comment, marked as answer]
Right now, it is a free text field. 

Do you think it should be enumerated? I'd be curious if it would be possible define the complete set of values.

**Reply by ghost** [reply]
I question the value of this field for a "individual user" who would download this file to search for a plan based billing code for a provider negotiated rate, instead it is geared more towards reporting and analytics. Being a machine readable file, my assumption is this field should contain a enumerated list of categories to allow it to be consumed and used for any reporting and analytics as we know free form text would not be useful for that. It really depends on its purpose and who the expected end users would be. From the file generation end, most file producers will be "hardcoding" a value in this field be it the same string they create or a value from a enumerated list.

**Reply by shaselton-usds** [reply]
Totally agree. 

That said, like with many of these elements, it is hard to nail down all the universe of permutations that are out there. There is the possibility of defining a set of values and then have a catch-all "OTHER" or something like that to select when there is a situation that isn't in the enumerated set. This has the downside of being rather ambiguous though. 

I think it would be interesting to do some analysis during the first year when these files are produced to determine the number a variant values that are used for the data element and respond accordingly to further implementation guidance.

**Reply by shaselton-usds** [reply]
@BobSyracuse -- Looking to clean up some of these discussions.  I'm going to close (assigned an accepted answer) this one; just let me know if you want to open it for further discussion.

**Reply by ghost** [reply]
@shaselton-usds All good! Thanks!
